### ISSUE TITLE
On fresh install any change to to Instruments just gives blank section

### ISSUE THREAD

**Issue by DaveyRance** [rcrum003/HiveControl#56]
I have just been through a number of attempts to get this up and running as the HiveTool.org image is just way to big and i dont need the desktop.

Downloaded 2017-01-11-raspbian-jessie-lite and booted.

Changed PI password, follow the instructions to curl the install.sh and then run. Install finish's fine and then asks to be rebooted. (Said no to bee counter) 

After install completes i am able to open the admin page and then set things on the Basic page. When i try to enable the scales on the instruments page things look good. Go to change the Weight Scale from Disabled to Enabled. The page refreshes with only the Settings - Instruments and the header. There are then  3 lines below the Settings - instruments full across the screen and no way to change the scale model. 

This happens for all of the settings on the page.

This happens on Chrome, IE 11 and Edge.

**Comment by rcrum003** [comment]
Can you send me a screenshot via email?

**Comment by DaveyRance** [comment]
Hi Ryan you should have the email all going well now.

**Comment by DaveyRance** [comment]
Before i installed Hivecontrol i updated the PI 

Enabled ssh under (raspi-config)
Then

sudo apt-get update && sudo apt-get -y dist-upgrade 
sudo apt-get autoremove
sudo apt-get clean

This is on a pi 3

**Comment by rcrum003** [comment]
We recommend downloading this image and following the instructions.

https://drive.google.com/open?id=0B8ZWirzPwqHfYzlDamlDZ0hxSE0 <https://drive.google.com/open?id=0B8ZWirzPwqHfYzlDamlDZ0hxSE0>

This is a smaller install file than the Hivetool image, which included desktop components.

Ryan

**Comment by DaveyRance** [comment]
I will try it again but that is a 3.4 gig image. :( i was hoping to make a minimum install and then add in MotionEye for the Camera so it is not needed to VNC in to the system to view the video. I will put a copy on a different SD and see if i can get that working. (Linux not my strongest suit)

**Comment by rcrum003** [comment]
Sorry, I gave you a bad link -

Here is the HiveControl only image, which is our latest (it deprecates the need for a desktop, and the old hivetool code base)

Install:
1. Download Image
2. Burn image to SD card, like any other image.
3. Boot, and then connect to the Webpage Admin console. (email me at rcrum[@]crumcasa[.].com) for the console password that is set.

Minimum SD Card Size = 8GB 
https://drive.google.com/file/d/0B8Vz5LCIukxGU1plU1hHMXBDSUU/view?usp=sharing <https://drive.google.com/file/d/0B8Vz5LCIukxGU1plU1hHMXBDSUU/view?usp=sharing>

Unique Features:
1. The image will autoresize the file system to the SD card size you have (so if you use a 32GB, we automatically allocate this space to use)
2. When booting, we set to DHCP, and it tells you the IP and link to manage it at the console

Note: We do not currently set to share with hivetool.org <http://hivetool.org/> because of the limitation of getting locked out if you aren’t a known hive (we have a proposed fix).
(We can the sharing in future releases and turn on during the update process)


Comments or concerns welcomed.
